modules/database/src/ioc/db/dbChannel.h

```diff
@@ -74,29 +77,30 @@ typedef struct dbChannel {
  */
 typedef db_field_log* (chPostEventFunc)(void *pvt, dbChannel *chan, db_field_log *pLog);
 
-/* Return values from chFilterIf->parse_* routines: */
+/** Return values from chFilterIf->parse_*- routines: */
 typedef enum {
     parse_stop, parse_continue
 } parse_result;
 
-/* These routines must be implemented by each filter plug-in */
+/** These routines must be implemented by each filter plug-in */
 typedef struct chFilterIf {
-    /* cleanup pointer passed to dbRegisterFilter().
-     * Called during DB shutdown
-     */
+/** 
+ * cleanup pointer passed to ::dbRegisterFilter ().
+ * Called during DB shutdown
+ */
```

Comment:
I think it would be nice to have a description of each of the parameters using the `\param` command